docs/reference/search/uri-request.asciidoc

```diff
@@ -1,10 +1,121 @@
 [[search-uri-request]]
 === URI Search
 
-A search request can be executed purely using a URI by providing request
+Sends a search request by using a URI with provided parameters.
+
+[source,js]
+--------------------------------------------------
+GET twitter/_search?q=user:kimchy
+--------------------------------------------------
+// CONSOLE
+// TEST[setup:twitter]
+
+
+[[search-uri-request-api-request]]
+==== {api-request-title}
+
+`GET {index}/_search?q={parameter}`
+
+
+[[search-uri-request-api-desc]]
+==== {api-description-title}
+
+A search request can be executed purely using a URI by providing request 
 parameters. Not all search options are exposed when executing a search
-using this mode, but it can be handy for quick "curl tests". Here is an
-example:
+using this mode, but it can be handy for quick "curl tests".
+
+
+[[search-uri-request-api-query-params]]
+==== {api-query-parms-title}
+
+`allow_partial_search_results`:: 
+  Set to `false` to return an overall failure if the request would produce 
+  partial results. Defaults to true, which will allow partial results in the 
+  case of timeouts or partial failures. This default can be controlled using the 
+  cluster-level setting `search.default_allow_partial_results`.
+
+`analyze_wildcard`:: 
+  Should wildcard and prefix queries be analyzed or not. Defaults to `false`.
+  
+`analyzer`:: 
+  The analyzer name to be used when analyzing the query string.
+
+`batched_reduce_size`:: 
+  The number of shard results that should be reduced at once on the coordinating 
+  node. This value should be used as a protection mechanism to reduce the memory 
+  overhead per search request if the potential number of shards in the request 
+  can be large.
+
+`default_operator`:: 
+  The default operator to be used, can be `AND` or `OR`. Defaults to `OR`.
+
+`df`:: 
+  The default field to use when no field prefix is defined within the query.
+
+`explain`:: 
+  For each hit, contain an explanation of how scoring of the hits was computed.
```

Comment:
We'll need the `(Optional/Required, datatype)` for all the following parameters.